[Bug] compile error caused by `mysql_scan_node.cpp` not being found

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/doris/issues?q=is%3Aissue) and found no similar issues.


### Version

master

### What's Wrong?

CMake Error at src/exec/CMakeLists.txt:82 (add_library):
  Cannot find source file:

    mysql_scan_node.cpp

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .mpp .m .M .mm .ixx .cppm .h
  .hh .h++ .hm .hpp .hxx .in .txx .f .F .for .f77 .f90 .f95 .f03 .hip .ispc


CMake Error at src/exec/CMakeLists.txt:82 (add_library):
  No SOURCES given to target: Exec

### What You Expected?

Compile successfully

### How to Reproduce?

_No response_

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/be/src/exec/CMakeLists.txt
+++ b/be/src/exec/CMakeLists.txt
@@ -65,13 +65,6 @@ set(EXEC_FILES
     table_connector.cpp
     schema_scanner.cpp
 )
-if (WITH_MYSQL)
-    set(EXEC_FILES
-        ${EXEC_FILES}
-        mysql_scan_node.cpp
-        mysql_scanner.cpp
-        )
-endif()
 
 if (WITH_LZO)
     set(EXEC_FILES ${EXEC_FILES}

--- a/be/src/vec/CMakeLists.txt
+++ b/be/src/vec/CMakeLists.txt
@@ -285,6 +285,13 @@ set(VEC_FILES
   exec/format/file_reader/new_plain_text_line_reader.cpp
   )
 
+if (WITH_MYSQL)
+    set(VEC_FILES
+            ${VEC_FILES}
+            exec/scan/mysql_scanner.cpp
+            )
+endif ()
+
 add_library(Vec STATIC
         ${VEC_FILES}
         )

